EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+2.	UTF-8 as encoding is used to provide flexibility to the values when written in different languages, also the RFC definition for JSON format specifies:<br>
+    * JSON text exchanged between systems that are not part of a closed ecosystem MUST be encoded using UTF-8<br>
+    * [RFC 8259](https://tools.ietf.org/html/rfc8259) - The JavaScript Object Notation (JSON) Data Interchange Format.
+
+
+3.	EPSG is one of the biggest registries for CRS definitions, that’s why the recommendation of use that one, however, as there are several formats allowed by the ISO 6709, depends strictly on the application on how to treat it.
+4.	ISO 6709 about decimal degrees: “For computer data interchange of latitude and longitude, this International Standard generally suggests that decimal degrees be used”
+
+## Backwards Compatibility
+
+One of the best aspects of this EIP is that it presents NO backwards incompatibilities. This is because any ERC-721 token that implements the Asset Metadata JSON does it optionally. This means that ERC-721 Tokens which do not have an Asset Metadata JSON will not be impacted. Similarly, any token which does implement it will continue to live regardless of the implementation of this EIP. Nonetheless, any non-compliant token can be made compliant with a simple metadata update with the location field and its corresponding attributes, as detailed in the [Technical Specification Section](#Specification). 
+
+This being said, there is one case where some confusion could arise. It would be when a token's metadata already has a "location" field which does not comply to this EIP. However, this particular case pertains more to the application level and how the NFT's metadata is being used. To conclude, this case does not generate backwards incompatibilities since no minted ERC-721 will be impacted but if a solution (as described earlier) were already to exist, a decision will need to be made as to which standard will be implemented.
+
+## Reference Implementation
+
+```json
+{ 
+  "name": "My Token",
+  "description": "Just a token",
+  "image": "https://images.blokchain.ey.com/0b1abca",
+  "location": {
+    "name": "My store",
+    "geolocation": {
+      "latitude": "+50.213",
+      "longitude": "-30.1322",
+      "altitude": "2000",
+      "crsid": "WGS84",
+    },
+    "address": {
+      "country": "USA",
+      "state": "New York",
+      "city": "New York",
+      "postal_code": "1312",
+      "postal_address": "5 Time Square",
+    }
+  }
+}
+```
+
+## Security Considerations
+
+There are no security implications for this proposal. This is because as we've described previously, this location standard exists within the Asset Metadata JSON of an ERC-721 and the only security implications, if any, would come from security implications in the [EIP-721](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-721.md).
+
+## Disclaimer
```

Comment:
```suggestion
## Disclaimer
```

Please remove this section, it is not a valid EIP section.

Reply:
Would it be possible to leave it there as long as the EIP is not in a Final stage? We'll take it out once all feedback is received.

Reply:
It will need to be removed before last call, but I am open to keeping the section until then.